tests/integration/targets/nxos_l2_interfaces/tests/common/replaced.yaml

```diff
@@ -1,24 +1,30 @@
 ---
 - debug:
-    msg: Start nxos_l2_interfaces replaced integration tests connection={{ ansible_connection
-      }}
+    msg: Start nxos_l2_interfaces replaced integration tests connection={{ ansible_connection }}
 
-- set_fact: test_int1="{{ nxos_int1 }}"
-
-- set_fact: test_int2="{{ nxos_int2 }}"
+- set_fact:
+    test_int1: "{{ nxos_int1 }}"
+    test_int2: "{{ nxos_int2 }}"
 
 - name: setup1
   ignore_errors: true
-  ansible.netcommon.cli_config: &id003
-    config: "default interface {{ test_int1 }}\ndefault interface {{ test_int2 }}\n"
+  cisco.nxos.nxos_config: &id003
+    lines:
+      - "default interface {{ test_int1 }}"
+      - "default interface {{ test_int2 }}"
 
 - block:
 
     - name: setup2
-      ansible.netcommon.cli_config:
-        config: "interface {{ test_int1 }}\n  switchport\n  switchport access vlan\
-          \ 5\ninterface {{ test_int2 }}\n  switchport\n  switchport trunk native\
-          \ vlan 15\n  switchport trunk allowed vlan 25-27\n"
+      cisco.nxos.nxos_config:
+        lines:
+          - "interface {{ test_int1 }}"
+          - "  switchport"
```

Comment:
This task needs to be split too. The l2_interfaces target is failing because of this.